public/app/features/dashboard-scene/scene/layout-rows/RowsLayoutManager.tsx

```diff
@@ -92,6 +102,7 @@ export class RowsLayoutManager extends SceneObjectBase<RowsLayoutManagerState> i
       name: 'Rows',
       description: 'Rows layout',
       id: 'rows-layout',
+      level: 1,
```

Comment:
I don't think level is a good filter prop, as rows could be useful inside Tabs layout for example. 

Maybe we can just filter out the same layout type? So inside DashboardLayoutSelector get the parent DashboardLayoutManager and it's id and filter out that

Reply:
The idea was that Tabs layout would have `level: 0`. That would result in Tabs supporting Rows but not vice-versa.

The alternative to this would be parent-based filtering but instead of having level, we would need to have something like `allowedNestedLayouts` in the descriptor since we need to filter out Tabs when having Rows layout selected at dashboard-level.

Reply:
ah I see, it's a bit confusing but it is a bit tricky :) 

I think it would also be good to technically support tabs inside a row (want to try it at least), and that would not work if rows was level 1 and tabs level 0.

Reply:
I thought about tabs in rows but it might create a little bit of confusion. Imagine having something like: `Tabs -> Rows -> Tabs -> ...` and the list can go for endless nesting.

Added a point on today's agenda for tabs inside rows.